spec/relativetimeformat.html

```diff
@@ -127,6 +127,7 @@ <h1>MakePartsList ( _pattern_, _unit_, _parts_ )</h1>
         1. Let _result_ be a new empty List.
         1. Let _beginIndex_ be ! Call(%StringProto_indexOf%, _pattern_, « "{", *0* »).
         1. Let _length_ be the number of elements in _pattern_.
+        1. Let _nextIndex_ be 0
```

Comment:
I suggest rewriting this algorithm with https://tc39.es/ecma402/#sec-partitionpattern , like I did in https://github.com/tc39/proposal-intl-list-format/pull/48.